.github/workflows/java8_integration_tests.yml

```diff
@@ -58,7 +58,6 @@ jobs:
           mkdir -p ~/.m2
           ALLUXIO_DOCKER_NO_TTY=true \
           ALLUXIO_DOCKER_GIT_CLEAN=true \
-          ALLUXIO_DOCKER_MVN_PROJECT_LIST=dora/tests \
```

Comment:
This is just a quick fix, the entire fix should be discussed further.

Reply:
PTAL👻! @Xenorith

Reply:
Because all integration tests are not run about three months, now so many fails appears in the workflows below.

Reply:
i think this is the correct fix for this file. but i think it also needs to be applied for the file at `.github/workflows/java8_integration_tests_ft.yml` and `.github/workflows/java8_integration_tests_webui.yml`

Reply:
applied

> i think this is the correct fix for this file. but i think it also needs to be applied for the file at `.github/workflows/java8_integration_tests_ft.yml` and `.github/workflows/java8_integration_tests_webui.yml`